Candlestick/Network/DataSource/KLines/KLinesRouter.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  KLinesRouter.swift
+//  Candlestick
+//
+//  Created by Ahmad Ragab on 20/07/2022.
+//
+
+import Foundation
+import Alamofire
+
+enum KLinesRouter: ApiBaseRouter {
+    case getKLines(String)
+}
+
+extension KLinesRouter {
+    
+    var path: String {
+        return NetworkConstants.kLines
+    }
+    
+    var parameters: [String : Any]? {
+        switch self {
+        case .getKLines(let symbol):
+            return [
+                "symbol" : symbol,
```

Comment:
It would be great if made the parameters as constants or enums to be changed in only one place